src/pt/mangaonline/src/eu/kanade/tachiyomi/extension/pt/mangaonline/MangaOnline.kt

```diff
@@ -0,0 +1,229 @@
+import rx.Observable
+import uy.kohesive.injekt.Injekt
+import uy.kohesive.injekt.api.get
+import java.text.SimpleDateFormat
+import java.util.Locale
+import java.util.concurrent.TimeUnit
+
+class MangaOnline() : ParsedHttpSource(), ConfigurableSource {
+    override val lang = "pt-BR"
+
+    override val supportsLatest = true
+
+    override val name = "Manga Online"
+
+    override val baseUrl = "https://mangaonline.biz"
+
+    private var genresSet: Set<Genre> = emptySet()
+
+    private val preferences: SharedPreferences =
+        Injekt.get<Application>().getSharedPreferences("source_$id", 0x0000)
+
+    override val client: OkHttpClient =
+        network.cloudflareClient.newBuilder()
+            .setRandomUserAgent(
+                preferences.getPrefUAType(),
+                preferences.getPrefCustomUA(),
+            )
+            .rateLimitHost(baseUrl.toHttpUrl(), 1, 2, TimeUnit.SECONDS)
+            .build()
+
+    override fun setupPreferenceScreen(screen: PreferenceScreen) {
+        addRandomUAPreferenceToScreen(screen)
+    }
+
+    override fun chapterFromElement(element: Element) = SChapter.create().apply {
+        name = element.selectFirst("a")!!.ownText()
+        date_upload = element.selectFirst("a span.date")?.ownText()!!.toDate()
+        setUrlWithoutDomain(element.selectFirst("a")!!.absUrl("href"))
+    }
+
+    override fun chapterListSelector() = "div.episodiotitle"
+
+    override fun imageUrlParse(document: Document) = ""
+
+    override fun latestUpdatesFromElement(element: Element) = SManga.create().apply {
+        title = element.selectFirst("h3 a")!!.ownText()
+            .replace("Capítulo\\s+([\\d.]+)".toRegex(), "")
+            .trim()
+
+        thumbnail_url = element.selectFirst("img")?.absUrl("src")
+
+        val mangaUrl = element.selectFirst("h3 a")!!.absUrl("href")
+            .replace("-capitulo-[\\d-]+".toRegex(), "")
+            .replace("capitulo", "manga")
+
+        setUrlWithoutDomain(mangaUrl)
+    }
+
+    override fun latestUpdatesNextPageSelector() = popularMangaNextPageSelector()
+
+    override fun latestUpdatesRequest(page: Int): Request = throw UnsupportedOperationException()
+
+    override fun fetchLatestUpdates(page: Int): Observable<MangasPage> {
+        val url = "$baseUrl/capitulo/page/$page".toHttpUrl().newBuilder()
+            .build()
+
+        return client
+            .newCall(GET(url, headers))
+            .asObservableSuccess()
+            .map {
+                val document = it.asJsoup()
+                val mangas = document
+                    .select(latestUpdatesSelector())
+                    .map { latestUpdatesFromElement(it) }
+                    .distinctBy { it.title }
+
+                /*
+                 NOTE: do not navigate to the next page because the manga will be duplicated.
+                 This source launches latest manga chapters unbundled.
+                 */
+                MangasPage(mangas, false)
+            }
+    }
```

Comment:
I did override the fetch, because the source adds new chapters separately, and if I use the regular request/parse methods it will display duplicate content.
![Captura de tela de 2024-02-27 13-34-21](https://github.com/keiyoushi/extensions-source/assets/156493704/f96cdc17-8dd7-4309-af7b-8523400ec832)

Reply:
yes you can filter in the parse method, something like this:
```
override fun latestUpdatesParse(response: Response): MangasPage {
    return super.latestUpdatesParse(response).distinctBy { it.title }
}

Reply:
Resolved: https://github.com/keiyoushi/extensions-source/pull/1566/commits/d986811a2a93570ad6050f5292721f4188f8c2b2